themes/cakephp/static/css/default.css

```diff
@@ -28,6 +29,13 @@
     margin: -50px 0 0; /* negative fixed header height */
 }
 
+.document-body {
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-weight: normal;
+    font-size: 15px;
+    letter-spacing: 0.02em;
```

Comment:
Letter spacing is not a good idea on floating text as it undos well-kerned font rendering.
https://developer.mozilla.org/de/docs/Web/CSS/font-kerning